spec/bing_translator_spec.rb

```diff
@@ -4,33 +4,18 @@
 
 require File.join(File.dirname(__FILE__), '..', 'lib', 'bing_translator')
 
-# Log all the tasks output, keeping RSpec interface clean while running
```

Comment:
Because:

- it's an anti-pattern (see https://12factor.net/logs)
- it requires rights access
- no logs are produced